### PR TITLE
GIX-2085: Use TokensTable in SignInAccounts

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

The accounts page for non-logged in users should be similar to the logged-in page.

Therefore, we use the TokensTable in SignInAccounts when flag is enabled.

# Changes

* Move the `MainWrapper` to the layout of the route accounts and apply it when flag is enabled.
* Add logic in the `SignInAccounts` to render different components depending on flag value.
* Use the `icpTokensListVisitors` when rendering `SignInAccounts` in the accounts route pages.

# Tests

* New SignInAccountsPo.
* Add methods to PO
* Add a test in the routes layout spec to check that token navigation is present or not depending on the flag.
* Add tests in the routes page to check that either tokens table or empty cards are present depending on the flag.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3810** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3810?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3810?utm_source=stack-comment).

File: frontend/src/lib/pages/SignInAccounts.svelte [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
-</script>
 
-<main data-tid="accounts-landing-page">
-  <SummaryUniverse />
+  export let userTokensData: UserTokenData[] = [];
+</script>
 
-  <div class="content">
+{#if $ENABLE_MY_TOKENS}
+  <div class="content" data-tid="accounts-landing-page">
```
Comment (lmuntaner) [inline frontend/src/lib/pages/SignInAccounts.svelte:15]: `main` is not needed here because it's in the Layout

**Comment by lmuntaner** [comment]
Flag not enabled:



Flag enabled

**Comment by lmuntaner** [comment]
@dskloetd @mstrasinskis please review

**Comment by peterpeterparker** [comment]
Unsolicited UX feedback: is "Projects" a new business term we want to introduce?

**Comment by lmuntaner** [comment]
> Unsolicited UX feedback: is "Projects" a new business term we want to introduce?

No, but thanks. That should change to "Accounts" here.

I'll do that in another PR because I fear it will be quite some changes in the table component. Issue: https://dfinity.atlassian.net/browse/GIX-2102

File: frontend/src/routes/(app)/(u)/(accounts)/+page.svelte [hunk 2]
```diff
@@ -3,12 +3,13 @@
...
   import TestIdWrapper from "$lib/components/common/TestIdWrapper.svelte";
+  import { icpTokensListVisitors } from "$lib/derived/icp-tokens-list-visitors.derived";
 </script>
 
 <TestIdWrapper testId="accounts-plus-page-component">
   {#if $authSignedInStore}
     <Accounts />
   {:else}
-    <SignInAccounts />
+    <SignInAccounts userTokensData={$icpTokensListVisitors} />
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/routes/(app)/(u)/(accounts)/+page.svelte:13]: Yes, the one in `tests/routes/app/accounts/page.spec`: "not logged in - tokens flag enabled  - renders tokens table for NNS accounts"

It renders the table only if there are rows. But let me make it more explicit with which tokens are expected.

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]
